modules/heat_conduction/test/tests/gap_heat_transfer_mortar/tests

```diff
@@ -248,6 +248,21 @@
                   'radiation between two blocks in 3D and use it as a reference for mortar-based '
                   'constraints.'
   []
+  [gap_heat_transfer_3d_hex20]
+    design = 'ModularGapConductanceConstraint.md'
+    issues = '#19286'
```

Comment:
Can these two items be upgraded to the top of the file (and remove repetitions for each test)?